packages/react-vapor/src/components/table-hoc/TableWithPrepend.tsx

```diff
@@ -9,7 +9,7 @@ export interface TableWithPrependProps extends ITableHOCOwnProps, Omit<React.HTM
     prepend?: React.ReactNode;
 }
 
-export const tableWithPrepend = (Component: React.ComponentClass<TableWithPrependProps>) => {
+export const tableWithPrepend = (Component: React.FC<TableWithPrependProps>) => {
```

Comment:
```suggestion
export const tableWithPrepend = (Component: React.FunctionComponent<TableWithPrependProps>) => {
```

Reply:
I received a suggestion on a previous PR a long time ago, we prefer to use `React.FunctionComponent` instead of `React.FC` since it's easier to understand.